docs/man/xrdp.ini.5.in

```diff
@@ -230,7 +230,7 @@ The following parameters can be used in the \fB[Logging]\fR section:
 
 .TP
 \fBLogFile\fR=\fI@localstatedir@/log/xrdp.log\fR
-This options contains the path to logfile. It can be either absolute or relative. If set to "-", log will go to stdout, which is useful for debugging\fR
+This options contains the path to logfile. It can be either absolute or relative. If set to \fB<stdout>\fR, log will go to stdout, which is useful for debugging\fR
```

Comment:
Pardon me for jumping in, but there's a (very) small issue here which hasn't been discussed.

At the moment, the log file is created with `O_APPEND` and `O_SYNC`. This has the useful effect of making sure that if an application forks and continues logging, the log messages are interleaved as you'd expect. There's a downside to this in that it really hits performance but that's a separate issue.

If we're logging to stdout, we lose the benefits of the interleaving, and it's possible with buffering for the main processes and the sub-processes to trample over each other's messages. If we're debugging, that's fine, but it's not something we should be using in production.

Consequently, in the above message I'd like to see *, which is useful for debugging* replaced with *. Use for debugging only.*

Does that make sense?

Reply:
Hi @metalefty

I've updated the PR according to to your comments

Reply:
Comment update above?

Reply:
I'm fine with that if you reflect Matt's comment.

Reply:
Hi,

sorry for delay. I've updated the documentation, as @matt335672 suggests